tools/misc/tpm2_checkquote.c

```diff
@@ -216,6 +219,37 @@ static bool pcrs_from_file(const char *pcr_file_path,
     return result;
 }
 
+static bool eventlog_from_file(tpm2_eventlog_ctx_t * ctx, const char *file_path) {
+
+    unsigned long size;
+
+    if (!files_get_file_size_path(file_path, &size)) {
+        return false;
+    }
+
+    if (!size) {
+        LOG_ERR("The eventlog file \"%s\" is empty", file_path);
+        return false;
+    }
+
+    uint8_t * eventlog = calloc(1, size);
+    if (!eventlog) {
+        LOG_ERR("OOM");
+        return false;
+    }
+
+    uint16_t size_tmp = size; // wtf uint16?
```

Comment:
TPM size field is uint16_t, so the lore was that we're always bounded by that. In this case, we're not. So it might be better just to use stdio directly and comment why.